task/06-conditions-n-loops-tasks.js

```diff
@@ -192,10 +195,13 @@ function findFirstSingleChar(str) {
  *
  */
 function getIntervalString(a, b, isStartIncluded, isEndIncluded) {
-    throw new Error('Not implemented');
+  let res = '';
+  isStartIncluded ? (res += '[') : (res += '(');
+  a < b ? (res += String(a) + ', ' + String(b)) : (res += String(b) + ', ' + String(a));
+  isEndIncluded ? (res += ']') : (res += ')');
```

Comment:
do not use assigments in ternary operators. That easy to miss. Just use palin old if statements or

`res = condition ? 'hello': 'bye'`

Reply:
According to your recomendation I used this construction res = condition ? 'hello': 'bye'instead of this one condition ? res = 'hello' : res = 'bye' , but still you marked it as a draft. Sure, i can just do it with 'if' but I just want to understand, why is it not correct now, because the current pice of code: res += (a < b) ? (String(a) + ', ' + String(b)) : (String(b) + ', ' + String(a)); is literally looks like res = condition ? 'hello': 'bye'